### ISSUE TITLE
update whitelist

### ISSUE THREAD

**Issue by tc80** [cdnjs/api-server#28]
We are missing `mjs` [here](https://api.cdnjs.com/whitelist)

**Comment by MattIPv4** [comment]
@tc80 Yeah, I think we need a deploy. This repo doesn't auto-deploy and I'm guessing it hasnt been deployed since I committed it.

**Comment by xtuc** [comment]
let's move the list of allowed extension on the cdnjs worker, we can share the list between the worker that serves the files (and block ext) and the internal API

**Comment by tc80** [comment]
good idea!
